mountpoint-s3/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## Unreleased
+## v1.8.0
```

Comment:
We should include a note on the bug fixes.

Add something like this after the new features section.

```markdown
### Other changes

* Support added for providing `external_id` when defining AWS profiles in a config file. ([#962](https://github.com/awslabs/mountpoint-s3/pull/962))
* Fix issue where empty environment variables for STS web identity credentials could cause segmentation fault. ([#246](https://github.com/awslabs/aws-c-auth/pull/246))

Reply:
done